.gitignore

```diff
@@ -1 +1,2 @@
-/node_modules
\ No newline at end of file
+node_modules
+dist
```

Comment:
A trailing slash will denote the fact it's a directory and not a file.

Reply:
done